channel-jira-cloud/src/main/java/com/synopsys/integration/alert/channel/jira/cloud/JiraCloudPropertiesFactory.java

```diff
@@ -44,28 +52,29 @@ public JiraCloudProperties createJiraProperties(FieldUtility fieldUtility) {
         String accessToken = fieldUtility.getStringOrNull(JiraCloudDescriptor.KEY_JIRA_ADMIN_API_TOKEN);
         boolean pluginCheckDisabled = fieldUtility.getBooleanOrFalse(JiraCloudDescriptor.KEY_JIRA_DISABLE_PLUGIN_CHECK);
         ProxyInfo proxy = proxyManager.createProxyInfoForHost(url);
-        return new JiraCloudProperties(url, accessToken, username, pluginCheckDisabled, proxy);
+
+        return new JiraCloudProperties(url, accessToken, username, pluginCheckDisabled, proxy, alertSSLContextManager.buildWithClientCertificate().orElse(null));
     }
 
     public JiraCloudProperties createJiraProperties(FieldModel fieldModel) {
         String url = fieldModel.getFieldValue(JiraCloudDescriptor.KEY_JIRA_URL).orElse("");
         String username = fieldModel.getFieldValue(JiraCloudDescriptor.KEY_JIRA_ADMIN_EMAIL_ADDRESS).orElse("");
         String accessToken = fieldModel.getFieldValueModel(JiraCloudDescriptor.KEY_JIRA_ADMIN_API_TOKEN)
-                                 .map(this::getAppropriateAccessToken)
-                                 .orElse("");
+            .map(this::getAppropriateAccessToken)
+            .orElse("");
         boolean pluginCheckDisabled = fieldModel.getFieldValue(JiraCloudDescriptor.KEY_JIRA_DISABLE_PLUGIN_CHECK)
-                                          .map(Boolean::parseBoolean)
-                                          .orElse(false);
+            .map(Boolean::parseBoolean)
+            .orElse(false);
 
         ProxyInfo proxy = proxyManager.createProxyInfoForHost(url);
-        return new JiraCloudProperties(url, accessToken, username, pluginCheckDisabled, proxy);
+        return new JiraCloudProperties(url, accessToken, username, pluginCheckDisabled, proxy, alertSSLContextManager.buildWithClientCertificate().orElse(null));
     }
 
     public JiraCloudProperties createJiraProperties() throws AlertConfigurationException {
         ConfigurationModel jiraServerGlobalConfig = configurationModelConfigurationAccessor.getConfigurationsByDescriptorKeyAndContext(channelKey, ConfigContextEnum.GLOBAL)
```

Comment:
Looks like there is another copied paste artifact if you can correct it in this PR as well